docs/source/conf.py

```diff
@@ -90,7 +91,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ['_build']
+exclude_patterns = ['_build', 'tests', 'setup']
```

Comment:
It seems necessary to rename `_build` to `build`

Reply:
Good catch. I just tested building locally and found that actually this `exclude_patterns` value isn't needed because I disabled the automatic `.rst`-file generation by removing `rtd_gen_docs`, and the other `.rst` files specify exact modules paths.

I changed it to exclude the whole `docs` directory, just in case the automatic module-finding is turned on again in the future.